fe/fe-core/src/test/java/com/starrocks/sql/analyzer/AnalyzeAlterTableStatementTest.java

```diff
@@ -1,3 +1,5 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
```

Comment:
Cannot add StarRocks license to old files

Reply:
this file is created by us. so I think it is OK.

Reply:
Yes, it looks like we added it, should use Elastic License 2.0